package.json

```diff
@@ -116,6 +123,8 @@
   },
   "dependencies": {
     "bootstrap-sass": "3.3.7",
```

Comment:
Move braces and core-js to devDependencies

Reply:
ok thx, moved both braces and core-js to devDependencies.